virttest/test_setup.py

```diff
@@ -1365,14 +1367,7 @@ def remove_driver(self, driver=None):
         if driver and driver != "mlx5_core":
             cmd = "modprobe -r %s" % driver
         if ARCH == 'ppc64le' and driver == 'mlx5_core':
-            pf_devices = self.get_pf_mlx()
-            logging.info("Mellanox PF devices '%s'", pf_devices)
             cmd = "rmmod mlx5_ib;modprobe -r mlx5_core;modprobe mlx5_ib"
-        elif ARCH == 'x86_64':
-            pf_devices = self.get_pf_ids()
-        for PF in pf_devices:
-            if not self.set_vf(PF):
-                return False
```

Comment:
Why do we need to disable vf first ? Is it a specific need for ppc+mlx5_core ?

Reply:
This is the old design,  keep it is recommended, as we don't have this card.

Reply:
@huangyum  i have test ppc + mlx5_core,  disable vf before remove pf is must for this condition.  Otherwise, vf can not be disabled when unload and load nic driver again, and it will caused generate vf failed after this.

Reply:
Thanks, updated. Please help review again.